setup.py

```diff
@@ -22,6 +22,7 @@ def readme():
         'djangorestframework',
         'drf-nested-routers',
         'django-filter',
+        'django-filters',
```

Comment:
@chrisbrantley tells me this isn't a required dependency, the `django-filter` package is the one actually being used (even though the code says `import django_filters`).

Chris, can you confirm? Also, you mentioned this was an optional feature of the library? Do we want to stick this under an `extras_require` setting in `setup.py`, and have them installed like `drf-jsonapi[with_filters]`?

Reply:
Correct. `django-filter` and `drf-yasg` are optional dependencies and not required for the framework to function. I love the idea of using `extras_require`.

`drf-nested-routers` is no longer needed at all.